CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [UNRELEASED] neptune 1.6.3
```

Comment:
Looks like someone already released that. Please, remove the `UNRELEASED` and move it to the `1.6.4` section.